interpreter.go

```diff
@@ -950,7 +950,8 @@ func jsonToValue(i *interpreter, v interface{}) (value, error) {
 	case bool:
 		return makeValueBoolean(v), nil
 	case int, int8, int16, int32, int64:
-		return makeDoubleCheck(i, v.(float64))
+		val, _ := v.(int64)
```

Comment:
I'm not sure why we need to convert the value to `int64`.

Is the following code wrong?

```go
return makeDoubleCheck(i, float64(v))
```

Reply:
I see.

`makeDoubleCheck(i, float64(v))` causes a compile error.

```
cannot convert v (variable of type interface{}) to type float64: need type assertion
```